selfdrive/controls/controlsd.py

```diff
@@ -339,7 +339,7 @@ def update_events(self, CS):
           self.events.add(EventName.cameraMalfunction)
         elif not self.sm.all_freq_ok(self.camera_packets):
           self.events.add(EventName.cameraFrameRate)
-    if not REPLAY and self.rk.lagging:
+    if not REPLAY and not SIMULATION and self.rk.lagging:
```

Comment:
this shouldn't be ignored. openpilot doesn't need much to run realtime

Reply:
done